### ISSUE TITLE
Spurious user file tag declaration warnings after .py copy and edit

### ISSUE THREAD

**Issue by codecat555** [talonvoice/talon#322]
`Talon Version: 0.1.5-256-gdb33`

Seeing repeated warnings after copying and modifying a .py file containing a tag declaration -

`WARNING tags: skipped because they have no matching declaration: (user.yyy)
`

To reproduce:
1. Under the user dir, create a .py file containing a tag declaration and a .talon file which activates that tag.
2. Copy the .py file to a new name and edit that copy to rename the tag.
3. The console starts spewing the warnings for the original tag name.
4.  Restart talon and verify that the warnings are no longer generated (indicating that there really isn't any problem with the files).

For example, I used this .py file...

```
from talon import Module

mod = Module()
mod.tag('yyy', desc='yyy')
```

...and this .talon file:

```
os: windows
app: ubuntu2004.exe
-

tag(): user.yyy
```
Note: the app name corresponds to one of my wsl instances...but I don't expect that matters here.

**Comment by lunixbochs** [comment]
This is actually a slightly different issue - if you define something in two modules and then unload one of them it deletes the thing. It's been low priority for me for now.

**Comment by codecat555** [comment]
Yeah, ok. It's really the symptom that I care about, personally, since I have to restart talon to stop it. Not a big deal, anyways - I'll just try to avoid that particular workflow.

**Comment by lunixbochs** [comment]
I think this is fixed in 0.1.5-486, let me know if you hit it again
